src/Symfony/Component/HttpKernel/Attribute/MapQueryParameter.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpKernel\Attribute;
+
+/**
+ * Can be used in a controller action to mark a parameter as a query parameter.
+ *
+ * public function homeAction(
```

Comment:
nitpicking here symfony no more "recommend" suffix `Action` see [doc](https://symfony.com/doc/current/controller.html#a-basic-controller)

appart from that this is very nice to have :)

Reply:
You're right, fixed.